### PR TITLE
Get Wolfram Alpha key from apiKeys instead of hardcoded config

### LINKED ISSUE #1320: Get Wolfram Alpha key from apiKeys instead of hardcoded config properties
**Actual Behaviour**

Currently, Wolfram Alpha key is taken from config.properties. For PR: https://github.com/fossasia/susi.ai/issues/2606, we require that the admin should be able to change key from config keys tab

**Expected Behaviour**

Get Wolfram Alpha key from apiKeys instead of hardcoded config properties

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1320

Changes: Currently, Wolfram Alpha key is taken from config.properties. For PR: fossasia/susi.ai#2606, we require that the admin should be able to change key from the config keys tab.

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @Orbiter please review

**Comment by fragm3** [comment]
@akshatnitd what could be the response, I need help with the fallback response here

**Comment by fragm3** [comment]
@akshatnitd made the changes

File: src/ai/susi/server/api/susi/ConsoleService.java [hunk 1]
```diff
@@ -152,23 +153,30 @@ public static void addGenericConsole(String serviceName, String serviceURL, Stri
             SusiThought json = new SusiThought();
             try {
                 String query = matcher.group(2);
-                String appid = DAO.getConfig("wolframalpha.appid", "");
+                JsonTray apiKeys = DAO.apiKeys;
+                JSONObject publicKeys = apiKeys.getJSONObject("public");
+                if(publicKeys.has("wolframalphaKey")) {
+                    String appid = publicKeys.getString("wolframalphaKey");
```
Comment (akshatnitd) [inline src/ai/susi/server/api/susi/ConsoleService.java]: Will it work? Won't appid show as not defined?

**Comment by fragm3** [comment]
@akshatnitd fixed code accordingly

**Pull request merged**